lib/oneview-sdk/resource/api200/logical_switch.rb

```diff
@@ -47,11 +47,10 @@ def create
       end
 
       # Reclaims the top-of-rack switches in a logical switch.
-      # @return [OneviewSDK::LogicalSwitch] self
-      def refresh_state!
+      # @return [Hash] Http response body with logical switch data.
```

Comment:
But, the @ client.response_handler (response) is called that waits for task finish and return the LogicalSwitch resource. This way occurs in many others methods of resources.